stream_alert/rule_processor/config.py

```diff
@@ -88,6 +88,14 @@ def _validate_config(config):
                 raise ConfigError(
                     'List of \'logs\' is empty for entity: {}'.format(entity))
 
+    # validate supported normalized types
+    supported_logs = [
+        'carbonblack', 'cloudwatch', 'cloudtrail', 'ghe', 'osquery', 'pan'
```

Comment:
a hardcoded list? Doesn't seem right... why not pull from the config?

Reply:
This code block is to validate if the log sources defined in the `types.json` supported. I will remove this code, looks redundant. We assume the log sources defined in `types.json` are supported.